class/Patchwork/Utf8.php

```diff
@@ -555,7 +555,7 @@ static function substr_replace($s, $replace, $start, $len = 2147483647)
     static function ucfirst($s)
     {
         $c = iconv_substr($s, 0, 1, 'UTF-8');
-        return static::ucwords($c) . substr($s, strlen($c));
+        return static::strtoupper($c) . substr($s, strlen($c));
```

Comment:
Why? Title case is not the same as upper case, and here, we need title case (see http://userguide.icu-project.org/transforms/casemappings)